doc/man7/EVP_RAND.pod

```diff
@@ -153,6 +153,11 @@ EVP_RAND_generate(<public>, ...) and
 EVP_RAND_generate(<private>, ...),
 respectively.
 
+When `[random] chain=0` is set in `openssl.cnf`, the public & private
+DRBG are not chained to primary, and instead use seed source
+directly. It is also possibly to configure OpenSSL to default to no
+chaining with -DOPENSSL_RAND_CHAIN=0.
```

Comment:
Considering the description in `RAND_priv_bytes()`

<pre>
RAND_priv_bytes() has the same semantics as RAND_bytes(). It is intended to be used for
generating values that should remain private. If using the default RAND_METHOD, <strong>this function
uses a separate "private" PRNG instance so that a compromise of the "public" PRNG instance will
not affect the secrecy of these private values</strong>, as described in RAND(7) and EVP_RAND(7).
</pre>

Should we describe more details? (e.g. `The primary DRBG will be returned as the public & private DRBG.`)

Otherwise, someone may not aware that there is no dedicated DRBGs for generating private/public-used random values after setting `OPENSSL_RAND_CHAIN=0`.

Reply:
The text as above seems to be inaccurate. Because compromising primary PRNG, compromises both public & private PRNGs, and there are APIs to directly add randomness to the primary PRNG =/ I'm not sure using separate instances is any different to reseeding, cause alternative implementation would simply reseed PRNG when switching to/from using it for public or private material.

Note I am considering to abandon this PR, and making a third attempt at this, which is backwards compatible with 3.4 and earlier FIPS providers, and doesn't break the 3.5 provider since https://github.com/openssl/openssl/pull/25415 got merged.

As in to initialize 3 DRBGs all using seed source directly. But there would be a different problem..... that adding entropy stops doing anything. Similar to what is now done for the FIPS provider.

Reply:
New PR at https://github.com/openssl/openssl/pull/25615